mico-core/src/main/java/io/github/ust/mico/core/dto/MicoApplicationStatusDTO.java

```diff
@@ -42,8 +45,67 @@
 public class MicoApplicationStatusDTO {
 
     /**
-     * List of deployment information of {@link MicoService}s, which belong to a {@link io.github.ust.mico.core.model.MicoApplication}.
+     * List of status information of {@link MicoService MicoServices}, which belong to a {@link MicoApplication}.
+     */
+    @ApiModelProperty(extensions = {@Extension(
+        name = CustomOpenApiExtentionsPlugin.X_MICO_CUSTOM_EXTENSION,
+        properties = {
+            @ExtensionProperty(name = "title", value = "ServiceStatus"),
```

Comment:
Adjust notation of value of name propery to conform to other DTOs, e.g., _Service Status_ (not _ServiceStatus_) or _Short Name_ (not _ShortName_).

Reply:
Check all other occurences.